helpers/browser-helper.js

```diff
@@ -26,4 +26,10 @@ module.exports = class BrowserHelper {
           });
     }
 
+    async setItemToLocalStorage(name, value) {
+        await page.evaluate((name, value) => {
+            localStorage.setItem(name, value)
+          }, name, value);
+    }
```

Comment:
As a suggestion to the code simplification you can rewrite the code above like this
```suggestion
     page.evaluate(localStorage.setItem, name, value);
```

Reply:
Well, `page.evaluate` won't work that way. It's tricky when dealing with function arguments. The way I wrote this is the best solution so far.